### PR TITLE
Initial draft of the getting started guide #2

### PR DESCRIPTION
**Pull request by scott-w**
I’ve copied over all the work I did in
https://github.com/marionettejs/backbone.marionette/issues/2448 with
the aim of modifying it to fit into the Marionette guide format.

I’d like to get some feedback and start a discussion about the
structure and content. I have a little more detail in #2.

### CODE REVIEW

**Comment by jdaudier** [comment]
Thanks @scott-w!  I'll look over your contribution this week.

**Comment by jdaudier** [comment]
Thanks @scott-w! We can iterate from here.

File: getting_started/first_app/events.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+  tagName: 'tr',
+  template: require('./templates/person.html'),
+
+  triggers: {
+    'click': 'click:item'
+  },
+
+  onClickItem: function(options) {
+    var counter = options.model.get('click_count');
+    options.model.set(counter + 1);
```
Comment (yethee) [inline getting_started/first_app/events.md]: Seems, should be

```js
  options.model.set('click_count', counter + 1);
```

Comment (scott-w) [inline getting_started/first_app/events.md]: Thanks for noticing, I'll get it sorted in my next PR :+1:

**Comment by jdaudier** [comment]
Thanks @yethee & @scott-w !

**Comment by jasonLaster** [comment]
yay @jdaudier, @scott-w! thanks!

**Pull request merged**